src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxNumber = 100;
```

Comment:
Такие поля называются константами. Они создаются один раз и не меняются в течение всей работы программы. Для таких полей есть определённые правила. Они должны быть static final и должны называться большими буквами
```suggestion
    private static final int MAX_NUMBER = 100;
```